etna/articles/models.py

```diff
@@ -226,3 +229,166 @@ def latest_items(self) -> Tuple["ArticlePage"]:
+                FieldPanel("image_library_link"),
+                FieldPanel("print_on_demand_link"),
+            ],
+        ),
+        FieldPanel("featured_article"),
+        TopicalPageMixin.get_time_periods_inlinepanel(),
+        TopicalPageMixin.get_topics_inlinepanel(),
+    ]
+
+    promote_panels = MetadataPageMixin.promote_panels + TeaserImageMixin.promote_panels
+
+    @cached_property
+    def gallery_items(self):
+        """
+        Used to access the page's 'gallery_images' for output. Makes use of
+        Django's select_related() and prefetch_related() to efficiently
+        prefetch image and rendition data from the database.
+        """
+        return (
+            self.gallery_images.all()
+            .select_related("image")
+            .prefetch_related("image__renditions")
+        )
+
+
+class PageGalleryImage(Orderable):
+    page = ParentalKey(Page, on_delete=models.CASCADE, related_name="gallery_images")
+    image = models.ForeignKey(
+        "wagtailimages.Image", on_delete=models.SET_NULL, null=True, related_name="+"
+    )
+    is_sensitive = models.BooleanField(
+        verbose_name="Is this image sensitive?",
+        default=False,
+    )
+    alt_text = models.CharField(
+        verbose_name=_("alternative text"),
+        max_length=100,
+        help_text=mark_safe(
+            'Alternative (alt) text describes images when they fail to load, and is read aloud by assistive technologies. Use a maximum of 100 characters to describe your image. <a href="https://html.spec.whatwg.org/multipage/images.html#alt" target="_blank">Check the guidance for tips on writing alt text</a>.'
+        ),
+    )
+    caption = RichTextField(
+        help_text="An optional caption, which will be displayed directly below the image. This could be used for image sources or for other useful metadata.",
+        blank=True,
+    )
+    transcription_header = models.CharField(
+        verbose_name=_("transcription header"),
+        max_length=50,
+        default="Transcript",
+        help_text=_("Header for the transcription (max length: 50 chars)."),
+    )
+    transcription_text = models.TextField(
+        verbose_name=_("transcription text"),
+        max_length=1500,
+        help_text=_("An optional transcription of the image (max length: 1500 chars),"),
+        blank=True,
+    )
+    translation_header = models.CharField(
+        verbose_name=_("translation header"),
+        max_length=50,
+        default="Translation",
+        help_text=_("Header for the translation (max length: 50 chars)"),
+    )
+    translation_text = models.TextField(
+        verbose_name=_("translation text"),
+        max_length=1500,
+        help_text=_(
+            "An optional translation of the transcription (max length: 1500 chars)."
+        ),
+        blank=True,
+    )
+
+    @property
+    def has_transcription(self):
+        return self.transcription_text != ""
+
+    @property
+    def has_translation(self):
```

Comment:
Question: What's the need for these properties? calling `instance.transcription_text` will have the same outcome as call `instance.has_translation`

Reply:
They'll help you write nicer/cleaner template code. 

`{% if gallery_image.has_transcription %}` > `{% if gallery_image.transcription_text != '' %}`